stable/jenkins/templates/_helpers.tpl

```diff
@@ -72,6 +72,14 @@ Returns configuration as code default config
 */}}
 {{- define "jenkins.casc.defaults" -}}
 jenkins:
+  {{- if and (eq .Values.master.enableXmlConfig false) (not (contains "authorizationStrategy:" (toYaml .Values.master.JCasC.configScripts))) }}
+  authorizationStrategy:
+    {{- tpl (toYaml .Values.master.JCasC.authorizationStrategy) . | nindent 4}}
+  {{- end }}
+  {{- if and (eq .Values.master.enableXmlConfig false) (not (contains "securityRealm:" (toYaml .Values.master.JCasC.configScripts))) }}
+  securityRealm:
+    {{- tpl (toYaml .Values.master.JCasC.securityRealm) . | nindent 4}}
+  {{- end }}
```

Comment:
Should we wrap `authorizationStrategy` and `securityRealm` with  `if eq .Values.master.enableXmlConfig false` to make it easier to read?

Reply:
I like this approach!

Reply:
> Should we wrap `authorizationStrategy` and `securityRealm` with `if eq .Values.master.enableXmlConfig false` to make it easier to read?

Yeah...good idea. I optimized with a $configScripts variable as well.